services/core/ActuatorAgent/tests/test_actuator_pubsub.py

```diff
@@ -560,6 +560,7 @@ def test_schedule_empty_task(publish_agent, cancel_schedules):
 
 
 @pytest.mark.actuator_pubsub
+@pytest.mark.xfail(reason="Request ids are now ignored.")
```

Comment:
Should probably remove tests that verify the identity instead of marking them xfail.